test/cpp_extensions/libtorch_agnostic_extension/setup.py

```diff
@@ -35,10 +36,16 @@ def get_extension():
         "cxx": ["-fdiagnostics-color=always"],
     }
 
+    extension = CppExtension
+    # allow including <cuda_runtime.h>
+    if torch.cuda.is_available():
+        extra_compile_args["cxx"].append("-DUSE_CUDA")
```

Comment:
Let's rename from USE_CUDA to LAE_USE_CUDA to avoid confusion with the USE_CUDA used in libtorch build